lessons/29-js-arrays/index.js

```diff
@@ -0,0 +1,19 @@
+const mySiblings=["Huswah", "Ridwan", "Abdurrahman","Toyyib"];
+const myParents=["Mother", "Dad"];
+const combinedArrays=mySiblings.concat(myParents)
+console.log(combinedArrays);
+combinedArrays.push("Ruby","Emerald");
+combinedArrays.reverse();
+console.log(myParents[1])
+myParents[0]=("Momma")
+const combinedArrays = ["Huswah", "Ridwan", "Abdurrahman","Toyyib","Momma","Dad"];
```

Comment:
careful, `combinedArrays` was declared above already, and actually 3 times in the same scope here. Javascript doesnt allow this. We'd need to give each of them different names.